SpatialGDK/Source/SpatialGDKFunctionalTests/Private/SpatialFunctionalTestFlowControllerSpawner.cpp

```diff
@@ -62,8 +62,6 @@ ASpatialFunctionalTestFlowController* SpatialFunctionalTestFlowControllerSpawner
 						   INVALID_FLOW_CONTROLLER_ID }; // by default have invalid id, Test Authority will set it to ensure uniqueness
 
 	FlowController->FinishSpawning(OwningTest->GetActorTransform(), true);
-	// TODO: Replace locking with custom LB strategy - UNR-3393
-	LockFlowControllerDelegations(FlowController);
```

Comment:
If we no longer lock them, we have to make sure the FlowController is setup in such a way that it allows migrations. I have looked at it briefly, and it seems like we just need to make `SpatialFunctionalTestStep CurrentStep;` Handover.

Alternative: still lock FlowControllers, but find a way such that it does not lock the playerController hierarchy. Haven't thought how yet.

Reply:
Well, I worked hard enough to have this property hold true, so I won't add a backdoor!
I'll look at the handover part.

Reply:
After trying to handover CurrentStep, I realized that Client's FlowController CurrentStep is never modified on the authoritative server (changing step just forward RPCs to the client), so handover would not make any difference.

Reply:
I'm talking out of memory, but in theory Clients would be fine with migrating (although this may trigger some issues if they constantly migrate) since what's important is that the FlowController's owner is the PlayerController, but I _think_ that you may need to ensure that the FlowController of each server has it's authority in the server that it wants to act, otherwise requests would be forwarded to another server.

Reply:
@nafonso Sure, Server's FlowControllers are still locked.